XMonad/Prompt.hs

```diff
@@ -228,6 +228,15 @@ class XPrompt t where
 
 data XPPosition = Top
                 | Bottom
+                -- | Prompt will be placed in the center horizontally and
+                --   in the certain place of screen vertically. If it's in the upper
+                --   part of the screen, completion window will be placed below(like
+                --   in 'Top') and otherwise above(like in 'Bottom')
+                | CenteredAt { xpCenterY :: Rational
+                             -- ^ Relative Y coordinate of center of the prompt.
```

Comment:
What does "relative Y coordinate" mean?  Is it an offset from the center, measured in pixels?  Is it a value between 0 and 1? etc?